src/sgp/Cursor_Control.cc

```diff
@@ -5,6 +5,9 @@
 #include "VObject.h"
 #include "Video.h"
 #include "VSurface.h"
+#include "ContentManager.h"
+#include "GameInstance.h"
+#include "policy/GamePolicy.h"
```

Comment:
Please move these to be in sorted order like the rest. See https://github.com/ja2-stracciatella/ja2-stracciatella/blob/master/CONTRIBUTING.md#axioms-of-style

Reply:
> Please move these to be in sorted order like the rest. See https://github.com/ja2-stracciatella/ja2-stracciatella/blob/master/CONTRIBUTING.md#axioms-of-style

I just looked at oher examples of `#include "policy/GamePolicy.h` and it seems to vary how it's included. What's most common is that it's separated by a linebreak from the main list of includes and it has ContentManager.h and GameInstance.h right above it.

Should I combine the new entries into the main list of includes, or do like most other files and add a linebreak before ContentManager.h?

Reply:
You can separate the subdir one, yeah.

Reply:
Okay, I'll just separate the subdir one.